src/foam/u2/view/UnstyledTableView.js

```diff
@@ -53,21 +54,28 @@ foam.CLASS({
     },
     {
       name: 'columns_',
+      documentation: `Note: filteredTableColumns was a property set in DAOController
```

Comment:
This shouldn't be intertwined within this expression like this. Let's use a reaction instead. Add this to the model:
```
reactions: [
  ['', 'propertyChange.columns_', 'updateFilteredTableColumns']
],
listeners: [
  {
    name: 'updateFilteredTableColumns',
    code: function() {
      this.filteredTableColumns = this.columns_.map(a => a.name);
    }
  }
]
```

Reply:
considering this should have been done weeks ago. Are you ok with me making a ticket and adjusting this after.?